manager.go

```diff
@@ -559,6 +565,19 @@ func (ovr *Overseer) StopAll(kill bool) {
 	ovr.access.Unlock()
```

Comment:
One like above: `ovr.setState(IDLE)`;
If we end up using `StopOverseer()` function, we can't switch the Overseer back into idle state, because we want it to be one-use only.